docs/get-started/sumo-logic-ui-new.md

```diff
@@ -0,0 +1,225 @@
+:::
+
+### Hide and show the left nav
+
+You can enlarge your working area by hiding the left-side navigation panel. To do this, click the hamburger menu icon.
+
+<img src={useBaseUrl('/img/get-started/hamburger.gif')} alt="hamburger.gif" width="350"/>
+
+To unhide it, click the hamburger menu icon again.
+
+
+## Using the top navigation bar
+
+### Access Apps, Admin settings, and Help
+
+From the top navigation bar, you can access (from left to right): Sumo Logic [Help](#get-help-docs-community-and-more), [App Catalog](apps-integrations.md), data [Configuration](/docs/send-data), [Administration](/docs/manage) settings, [Notifications](account-settings-preferences.md), and personal [Preferences](account-settings-preferences.md).
+
+<img src={useBaseUrl('/img/get-started/admin-config.png')} alt="admin-config.png" width="350"/>
+
+### Manage your personal account preferences
+
+You can manage your personal account settings from the **Preferences** page. These settings apply only to your account. Changes you make to your preferences take effect the next time you sign in, not during the current session.
+
+To manage your personal Sumo account preferences:
+
+1. From the top navigation bar, click the person icon, and then from the dropdown, select **Preferences**.<br/><img src={useBaseUrl('img/get-started/acct-pref.png')} alt="img/get-started/acct-pref.png" width="350"/>
+1. In the Preferences page, you can modify the following settings:
+    * **My Profile**. Username and password.
+    * **My Access Keys**. Add, edit, and remove access keys.
+    * **My Preferences**. Your account session settings.
+
+For more information, see [Account Preferences and Credentials](account-settings-preferences.md).
+
+### Get Help: Docs, community, and more
+
+There are a number of places where you can get the answers to questions you have:
+
+* Check out our [Release Notes](/docs/release-notes)
+* Search [Sumo Docs](https://help.sumologic.com/)
+* Visit the **Learn Page** in the Sumo Logic UI
+* Post a question on the [Sumo Logic Community](https://support.sumologic.com/hc/en-us/community/topics)
+* Contact [Support](https://support.sumologic.com/)
+* Join our [Sumo Logic Slack](https://sumodojo.slack.com/) channel
+
+:::tip
+See [Help Resources and Contacts](/docs/get-started/help) for full information.
+:::
+
+## Administrator tasks
+
+### Manage data collection, data settings, and alerts
+
+Sumo Logic Administrators (Admins) are responsible for managing data collection, data settings, and alerts for their organization. You must have Sumo Logic Admin role privileges to perform these tasks.
+
+To manage data in Sumo Logic, do the following:
+
+1. From the top-right navigation bar, click **Configuration**.<br/><img src={useBaseUrl('img/get-started/config.png')} alt="img/get-started/config.png" width="400"/>
+1. Choose from the following, as needed:
```

Comment:
For the items in this list, should you show them exactly as they appear in the UI? It would make it easier for readers to match them with what they see in the UI.

For example:
* Collection. [Collection](/docs/send-data/collection/), [OpenTelemetry Collection](/docs/send-data/opentelemetry-collector/), [Status](/docs/manage/ingestion-volume/collection-status-page/), [Ingest](/docs/manage/ingestion-volume/ingest-budgets/), Archive, [Data Archiving](h/docs/manage/data-archiving/).
* Logs. [Fields](/docs/manage/fields/), [Field Extraction Rules](/docs/manage/field-extractions/), [Parsers](/docs/search/search-query-language/parse-operators/), [Partitions](/docs/manage/partitions-data-tiers/), [Scheduled Views](/docs/manage/scheduled-views/), [Data Forwarding](/docs/manage/data-forwarding/), [Threat Intelligence](/docs/platform-services/threat-intelligence-indicators/).
* Metrics. [Metrics Rules](/docs/metrics/metric-rules-editor/), [Logs-to-Metrics](/docs/metrics/logs-to-metrics/), [Metrics Transformation Rules](/docs/metrics/metrics-transformation-rules/).
* Cloud SIEM Integrations. [Ingest Mappings](/docs/cse/ingestion/sumo-logic-ingest-mapping/), [Log Mappings](/docs/cse/schema/create-structured-log-mapping/), [Context Actions](/docs/cse/administration/create-cse-context-actions/), [Actions](/docs/cse/administration/create-cse-actions/), [Enrichment](/docs/cse/integrations/enrichments-and-indicators/), [Automation](/docs/cse/automation/).
* Cloud SIEM Entities. [Groups](/docs/cse/records-signals-entities-insights/create-an-entity-group/), [Normalization](/docs/cse/records-signals-entities-insights/configure-entity-lookup-table/), [Custom Types](/docs/cse/records-signals-entities-insights/create-custom-entity-type/), [Criticality](/docs/cse/records-signals-entities-insights/entity-criticality/).
* Cloud SIEM Workflow. [Insight Detection](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold/), [Insight Statuses](/docs/cse/administration/manage-custom-insight-statuses/), [Insight Resolutions](/docs/cse/administration/manage-custom-insight-resolutions/), [Tag Schemas](/docs/cse/administration/create-a-custom-tag-schema/).
* Cloud SOAR Configurations. Manage [Incidents](/docs/cloud-soar/incidents-triage/), [Fields](/docs/cloud-soar/incidents-triage/#custom-fields), [Incident Labels](/docs/cloud-soar/global-functions-menu/#incident-label).

Reply:
fixed!